docker/env/version

```diff
@@ -1 +1 @@
-0.57
+0.58
```

Comment:
bump it to 0.59, 0.58 is merged already today

Reply:
fixed. and rebased on latest master